crates/qasm2/src/lex.rs

```diff
@@ -337,7 +337,7 @@ pub struct TokenStream {
     /// backing file or other named resource.
     pub filename: std::ffi::OsString,
     strict: bool,
-    source: Box<dyn std::io::BufRead + Send>,
+    source: Box<dyn std::io::BufRead + Send + Sync>,
```

Comment:
Is the `Sync` bound now required because of the free-threading support?

Reply:
Yeah, it was a requirement that the members of a pyclass were all Sync